.github/workflows/cleanup.yaml

```
@@ -2,13 +2,14 @@ name: Cleanup
 
 on:
   schedule:
-    # Runs on 19:00 JST every day, note that cron syntax applied as UTC
-    - cron: '0 10 * * *'
+    # Runs on 23:30 JST every day, note that cron syntax applied as UTC
+    - cron: "30 14 * * *"
   workflow_dispatch:
 
-permission:
+permissions:
   packages: delete
```

actionlint (1.7.4)

"delete" is invalid for permission of scope "packages". available values are "read", "write" or "none"

(permissions)